### PR TITLE
HPCC-16790 BS ES HR PT-BR SR ZH Translations

### PR DESCRIPTION
**Pull request by GordonSmith**
Fixes HPCC-16790

Signed-off-by: Gordon Smith <gordonjsmith@gmail.com>

### CODE REVIEW

**Comment by GordonSmith** [comment]
@richardkchapman @miguelvazq @dehilsterlexis @emuharemagic @rpastrana @mayx Please Review

File: esp/src/eclwatch/nls/es/hpcc.js [hunk 1]
```diff
@@ -760,6 +762,7 @@ define(
     XRef: "Referencias cruzadas",
     Year: "Año",
     YouAreAboutToDeleteBinding: "Está a punto de eliminar esta conexión. Desea continuar?",
+    YouAreAboutToRemoveUserFrom: "Usted esta a punto de quitar un usuario(s) de este grupo. ¿Desea continuar?",
```
Comment (rpastrana) [inline esp/src/eclwatch/nls/es/hpcc.js]: Minor:
Usted esta a punto de quitar un(os) usuario(s) de este grupo. ¿Desea continuar?",

**Review by emuharemagic — APPROVED** [review]

**Comment by GordonSmith** [comment]
@richardkchapman, @rpastrana  change made as per comment

**Comment by rpastrana** [comment]
@GordonSmith ES looks good.

**Comment by mayx** [comment]
@GordonSmith  Chinese looks good.

**Comment by mayx** [comment]
After reviewing the whole hpcc.js file, I’d like to tweak the Chinese translation a bit to conform to the overall convention:

“请选择要将用户添加到的用户组”
“请选择要添加的用户”
“你即将从此用户组中删除用户。你要继续吗？”

**Comment by mayx** [comment]
The difference is very minor and won't cause any confusion so there's no need to change now as the build is already out.

**Comment by richardkchapman** [comment]
We can change for 6.2.2 if you like?

**Comment by mayx** [comment]
Sure, that's what Gordon suggested also.

**Pull request merged**